amundsen-kube-helm/templates/helm/values.yaml

```diff
@@ -30,6 +30,58 @@ tolerations: []
 # podAnnotations -- amundsen application wide configuration of podAnnotations. This applies to search, metadata, frontend and neo4j. Elasticsearch has it's own configuation properties for this. [ref](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
 podAnnotations: {}
 
+# Optional Flask App wrapper to wrap your Amundsen Microservices
+# Important: If you are using oidc.enabled=true, then you don't need to set this explicitly,
+# as this will be set for you.
+flaskApp:
+  #  flaskApp.module -- Any custom flask module you may need to implement as a wrapper
+  module: ""
+  #  flaskApp.class -- The class name within the flaskApp.module
+  class: ""
+
+# To enable OIDC,
+#     - please make sure set frontend.config.class: amundsen_application.oidc_config.OidcConfig
+# Following variables must be set for each service under the oidc.frontend, oidc.metadata and oidc.search.
+# - FLASK_OIDC_CLIENT_ID
+# - FLASK_OIDC_CLIENT_SECRET
+# More Information: https://github.com/verdan/flaskoidc#configurations
+oidc:
+  # oidc.enabled -- flag to enable/disable the OIDC.
+  # Once enabled,
+  #   - everything under oidc.configs will be parsed
+  #   - flaskApp.module will be set as 'flaskoidc'
+  #   - flaskApp.class will be set as 'FlaskOIDC'
+  enabled: true
+  frontend:
+    client_id: ""
+    client_secret: ""
+  metadata:
+    client_id: ""
+    client_secret: ""
```

Comment:
I would only leave comment not to paste secret directly here but pass it from env variables when running `helm` like 
```
`--set oidc.metadata.client_secret=$OIDC_METADATA_CLIENT_SECRET`
```